corehq/motech/repeaters/dbaccessors.py

```diff
@@ -227,34 +227,6 @@ def get_sql_repeat_records_by_payload_id(domain, payload_id):
             .all())
 
 
-def get_repeaters_by_domain(domain):
```

Comment:
Curious why these were moved? This file will eventually be deleted, and these will all go away since they are Couch utilities, right? If that's true, moving them around seems like extra/unnecessary churn on the road to removing them completely.

I think the ideal is to re-implement all dbaccessor functions as Django model manager methods. Is that the direction this is headed?

Reply:
Ideally, we would implement dbaccessors in SQL but they were used only in the migration code. So I moved it to migration util. The code would be removed when the migration is removed and thereby removing the dbaccessors which can be replaced by simple SQL queries.

There is one management command where one of the dbaccessors was used which would be updated in the next PR.